controllers/cards.js

```diff
@@ -1,36 +1,43 @@
+const mongoose = require('mongoose');
 const Card = require('../models/card');
 
-module.exports.getAllCards = (req, res) => {
+const { ObjectId } = mongoose.Types;
+
+const NotFoundError = require('../errors/error_not_found');
+const Error403 = require('../errors/error_403');
+
+module.exports.getAllCards = (req, res, next) => {
   Card.find({})
-    .then((card) => {
-      if (card.length === 0) {
-        return res.status(404).send({ message: 'База данных cards пуста! ' });
-      }
-      return res.send({ data: card });
-    })
-    .catch((error) => res.status(500).send({ message: error.message }));
+    .then((card) => res.send({ data: card }))
+    .catch(next);
 };
 
-module.exports.createCard = (req, res) => {
-  const owner = req.user._id;
+module.exports.createCard = (req, res, next) => {
+  /* const owner = req.user._id; */
   const { name, link } = req.body;
-  Card.create({ name, link, owner })
+  Card.create({ name, link, owner: req.user._id })
     .then((card) => res.send({ data: card }))
-    .catch(() => res.status(500).send({ message: 'Не удается создать карточку' }));
+    .catch(next);
 };
 
-module.exports.deleteCard = (req, res) => {
+module.exports.deleteCard = (req, res, next) => {
   const { cardId } = req.params;
-  const ownerId = req.user._id;
-  Card.findById(cardId)
+  if (!ObjectId.isValid(cardId)) {
+    return (new NotFoundError('not found'));
+  }
+  Card.findById(req.params.cardId)
```

Comment:
Нужно исправить: ошибки линтинга.
```bash
/Users/jmi/work/ya-reviews/15_sprint/controllers/cards.js
  23:46  error  Expected to return a value at the end of arrow function  consistent-return
```
```suggestion
  return Card.findById(req.params.cardId)
```